### DISCUSSION TITLE
I have a strange problem with this domain

### DISCUSSION THREAD

**Discussion by majedmz** [AdguardTeam/AdGuardHome#6917] [Q&A]
I have a strange problem with this domain
https://schools.madrasati.sa/
  It is not on the blocked list, but it is blocked and I cannot access it at all
What do I do? Is there help here?

**Comment by amrmzr** [comment]
Add your blocked website in custom filtering like below given to unblock it
e.g
@@||www.schools.co^$important

**Reply by majedmz** [reply]
Thankyou
I added to allow list but not working
problem is still

**Reply by amrmzr** [reply]
I can't access it either.
can you confirm if it is geographically blocked or not using VPN?
and I tried to check if its Host is up using Nmap & its NOT, maybe wrong address.

thanks!

**Comment by zefr0x** [comment]
I found such problem some time ago with the Cloudflare DNS **(not in AdGuard)**, where the domain was using some DNS-level load balancing thing I guess, and Cloudflare didn't support having access to or something.

I solved this by redirecting any query with this domain to Google's DNS server and it works.

I guess it should also work with other DNS providers so go and test it, try to avoid using Google's.

**Comment by Dynamic5912** [comment]
> I found such problem some time ago with the Cloudflare DNS **(not in AdGuard)**, where the domain was using some DNS-level load balancing thing I guess, and Cloudflare didn't support having access to or something.
> 
> 
> 
> I solved this by redirecting any query with this domain to Google's DNS server and it works.
> 
> 
> 
> I guess it should also work with other DNS providers so go and test it, try to avoid using Google's.

This is a 2 year old post...
